package.json

```diff
@@ -20,7 +20,9 @@
     "build-rules": "node rules/build.js && cp rules/rules.json addon/",
     "version": "node update_version.js && git add addon/manifest.json",
     "vendor-copy": "mkdir -p addon/vendor && cp node_modules/mocha/mocha.* addon/vendor/ && cp node_modules/chai/chai.js addon/vendor/",
-    "prepublish": "npm run build-rules && npm run bundle"
+    "prepublish": "npm run build-rules && npm run bundle",
+    "use-mv2": "cp addon/manifest.firefox.json addon/manifest.json",
+    "use-mv3": "git checkout -- addon/manifest.json"
```

Comment:
Depending on the git state could be inconvenient here: annoying if you want to edit the manifest file, and copying files like this breaks idempotency of the npm scripts.
Shall we just add a new build target in the rollup recipe? So we have nice separate bundles for chrome and firefox

Reply:
You mean duplicate the `addon` dir and have `addon-mv2` with the mv2 code in? Or get rollup to handle copying the correct manifest version in?

Reply:
Since the code can handle both versions, I guess we don't have to copy the source files. So we can just put manifests in 2 separate folders and make rollup produce bundles in both of them.
Or, just change the manifests themselves to point to the JS in the same external directory, but I'm not sure if it's allowed.